package.json

```diff
@@ -40,7 +40,7 @@
     "dist/**"
   ],
   "scripts": {
-    "build": "tsc -p tsconfig-cjs.json && tsc -p tsconfig-es.json",
+    "build": "tsc -p tsconfig-cjs.json && tsc -p tsconfig-es.json && renamer --find mjs --replace js ./dist/cjs/*",
```

Comment:
To make things a bit easier I think you could use https://github.com/egoist/tsup

Reply:
Nice I'll check that out and see how to use that

Reply:
Done! Not sure if it's a breaking change to now just have index.js and all the other files gone. The tsc method kept all of the files the same if this is a breaking change.